site/gatsby-site/src/pages/apps/reports.js

```diff
@@ -451,12 +450,7 @@ export default function Incidents(props) {
         <title>Incident List</title>
       </AiidHelmet>
 
-      {loading && (
-        <div className="p-4 flex justify-center align-items-center gap-2">
-          <Spinner />
-          <Trans>Fetching Reports...</Trans>
-        </div>
-      )}
+      {loading && <ListSkeleton />}
```

Comment:
The left edge seems to get cut off here.

![image](https://user-images.githubusercontent.com/25443411/205670652-966233b3-f1e5-488d-a4aa-e92a68224690.png)